src/components/MapSlideUpSheet.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import {
+  View,
+  StyleSheet,
+  Button,
+  Text,
+  Dimensions,
+  TouchableOpacity,
+} from "react-native";
+import SlidingUpPanel from "rn-sliding-up-panel";
+import SearchBar from "@components/SearchBarView";
+import SearchView from "@components/SearchView";
+import SearchResultSorter from "@components/SortSearchResults/SearchResultSorter";
+
+const { height } = Dimensions.get("window");
+
+class MapSlideUpSheet extends React.Component {
```

Comment:
We don't use class based components, only functional ones.

Reply:
What's the problem with a class based component?

Reply:
^^ this does make sense for consistency and said we would stick to functions at the start but idm if it's a pain to change.